pkg/tuple/tuple.go

```diff
@@ -293,6 +293,16 @@ func MustFromRelationship(r *v1.Relationship) *core.RelationTuple {
 	return FromRelationship(r)
 }
 
+// MustFromRelationships converts a slice of Relationship's into a slice of RelationTuple's.
+func MustFromRelationships(rels []*v1.Relationship) []*core.RelationTuple {
```

Comment:
I think you meant to use this in `loader.go` right?

Reply:
Can't; we need to build updates in there too, so I just added the method

Reply:
What I actually suggested in my review was to create a method that took "relationship" and returned touch updates. You can have an argument in the method with the kind of update operation, so it can also be CREATE or DELETE.

Reply:
We need to add it to two slices though, so that won't work either. As discussed offline, leaving as-is